seeds/addAuthors.js

```diff
@@ -39,46 +49,53 @@ const createAllFakeAuthors = (numberAuthors) => {
 
 // const allAuthors = createAllFakeAuthors(1000000);
 
-exports.seed = (knex) => {
-  return knex('authors').del()
-    .then(() => {
-      return knex.batchInsert('authors', createAllFakeAuthors(1000000), 1000)
-        .then(() => {
-          return knex.batchInsert('authors', createAllFakeAuthors(1000000), 1000)
-            .then(() => {
-              return knex.batchInsert('authors', createAllFakeAuthors(1000000), 1000)
-                .then(() => {
-                  return knex.batchInsert('authors', createAllFakeAuthors(1000000), 1000)
-                    .then(() => {
-                      return knex.batchInsert('authors', createAllFakeAuthors(1000000), 1000)
-                        .then(() => {
-                          return knex.batchInsert('authors', createAllFakeAuthors(1000000), 1000)
-                            .then(() => {
-                              return knex.batchInsert('authors', createAllFakeAuthors(1000000), 1000)
-                                .then(() => {
-                                  return knex.batchInsert('authors', createAllFakeAuthors(1000000), 1000)
-                                    .then(() => {
-                                      return knex.batchInsert('authors', createAllFakeAuthors(1000000), 1000)
-                                        .then(() => {
-                                          return knex.batchInsert('authors', createAllFakeAuthors(1000000), 1000)
-                                            .catch(err => console.log(err));
-                                        })
-                                        .catch(err => console.log(err));
-                                    })
-                                    .catch(err => console.log(err));
-                                })
-                                .catch(err => console.log(err));
-                            })
-                            .catch(err => console.log(err));
-                        })
-                        .catch(err => console.log(err));
-                    })
-                    .catch(err => console.log(err));
-                })
-                .catch(err => console.log(err));
-            })
-            .catch(err => console.log(err));
-        })
-        .catch(err => console.log(err));
-    });
-};
\ No newline at end of file
+// exports.seed = (knex) => {
```

Comment:
You mentioned this, but just a reminder to get rid of these comments if you don't need them 🙃